extensions/tracing.md

```diff
@@ -24,3 +24,15 @@ The second dependency is [Jaeger](https://www.jaegertracing.io/) implementation
 
 There are other supported tracers that can be used: LightStep, Instana, Apache SkyWalking, Datadog, Wavefront by VMware, Elastic APM and many more.
 
+## Configuring the extension
+The extension could be disabled using the property `axon.extension.tracing.enabled` (default=TRUE). This will give you the possibility to turn it off for a certain environment.
```

Comment:
```suggestion
The extension can be disabled setting the property `axon.extension.tracing.enabled` to `false` (default=`true`). This will give you the possibility to turn it off when needed (e.g.: for a certain environment).
```